content/2021-08-19-GSoC21.rst

```diff
@@ -0,0 +1,101 @@
+GSoC’21 Improve performance through the use of Pythran
+######################################################
+
+:date: 2021-08-19
+:category: examples
+:lang: en
+:authors: Xingyu Liu
+:summary: This project is about using Pythran to accelerate algorithms in SciPy and 
+          writing benchmarks for the algorithms. Let's look into the details of the project.
+
+
+
+Project Overview
+================
+There are a lot of algorithms in scipy that use Cython to improve 
+the performance of code that would be too slow as pure Python, 
+e.g. algorithms in ``scipy.spatial``, ``scipy.stats`` and ``scipy.optimize``, etc. 
+Recently, SciPy added experimental support for Pythran, 
+to make it easier to accelerate Python code. 
+Compared with Cython, Pythran is more readable and even faster. 
+Furthermore, SciPy uses Airspeed Velocity for performance benchmarking. 
+Therefore, our project includes:
+
+
+* Accelerating SciPy algorithms with Pythran.
+* Writing benchmarks for the algorithms in SciPy
+* Find and solve potential issues in Pythran
+
+
+My full proposal can be accessed `here <https://docs.google.com/document/d/1nM7dYbmModiukQw-sSOVGz6t5S6HC0VVWucYadI_aMQ/edit?usp=sharing>`_.
+
+
+What I have done
+================
+
+**Pull Requests**:
```

Comment:
Do you want to split this list in contributions to SciPy and to Pythran? The latter may be nice to separate out and say a few words about. I think that made your project more interesting and challenging overall, so it'd also be good to show that.